public/src/components/channelManagement/bannerDesigns/BannerDesignForm.tsx

```diff
@@ -127,6 +166,43 @@ const BannerDesignForm: React.FC<Props> = ({
           />
         </div>
       </div>
+      <div className={classes.sectionContainer}>
+        <Typography variant={'h3'} className={classes.sectionHeader}>
+          Basic Colours
+        </Typography>
+        <div>
+          <TextField
```

Comment:
we discussed earlier the possibility of creating a component that validates _and_ transforms the string into a `HexColour`, but it's fine to ship as it is and refactor later

Reply:
I'm definitely pro this suggestion (both the idea and to tackle in a separate PR)!